index.js

```diff
@@ -991,3 +1023,20 @@ function updateProposalsCount (count) {
   var numberField = document.querySelector('#proposals-count-number')
   numberField.innerText = (count.toString() + ' proposal' + (count !== 1 ? 's' : ''))
 }
+
+function updatedFilterStatus () {
+  var labels = [].concat.apply([], document.querySelector('#filter-options').querySelectorAll('label'))
```

Comment:
I think the following are equivalent:

* `document.querySelector('#filter-options').querySelectorAll('label')`

* `document.querySelectorAll('#filter-options label')`

Reply:
@benrimmington Exactly

Reply:
Fixed it